src/content/tools/mcp-server.md

```diff
@@ -7,7 +7,7 @@ description: >-
   AI-assistant clients and agents.
 ---
 
-The Dart MCP server exposes Dart (and Flutter)
+The Dart MCP server exposes Dart and Flutter
```

Comment:
```suggestion
The Dart MCP server exposes Dart (and Flutter)
```

Let's keep (and Flutter) in parenthesis to stress that this guide is for Dart. We should try to strip out as much Flutter-specific content as possible and add that content to the Flutter docs.